src/docs/torque.md

```diff
@@ -202,7 +202,7 @@ type int31 extends int32 generates 'TNode<Int32T>' constexpr 'int31_t';
 Union types express that a value belongs to one of several possible types. We only allow union types for tagged values, because they can be distinguished at runtime using the map pointer. For example, JavaScript numbers are either Smi values or allocated HeapNumber objects.
 
 ```torque
-type Number = Smi | HeapNumber;
+type Number = Smi|HeapNumber;
```

Comment:
Is this intentional, or should we fix the formatter to add the spaces? They kind of looked nice. cc @tebbi

Reply:
No, this isn't intentional, but the JS formatter that we use somehow doesn't understand this syntax and we'd have to fix it/work around it explicitly.

Reply:
Fixed by https://chromium-review.googlesource.com/c/v8/v8/+/1297949. Thanks, @dannoc!